vault/resource_pki_secret_backend_issuer.go

```diff
@@ -164,6 +184,15 @@ func pkiSecretBackendIssuerUpdate(ctx context.Context, d *schema.ResourceData, m
 		consts.FieldEnableAIAURLTemplating,
 	}
 
+	if supportPkiCertVerifyDisableChecksFields(meta) {
```

Comment:
nit: What would you think of collapsing the three blocks that add these fields into a dedicated function that takes meta and the slice and returns a modified slice with the values added if required? 

I'm not sure if it's clearer to see the fields being added inline vs having one function that all the fields are added and removing the duplication/chance of missing a field.